src/potree.ts

```diff
@@ -47,12 +49,16 @@ export class Potree implements IPotree {
   features = FEATURES;
   lru = new LRU(this._pointBudget);
 
-  loadPointCloud(
-    url: string,
-    getUrl: GetUrlFn,
-    xhrRequest = (input: RequestInfo, init?: RequestInit) => fetch(input, init),
-  ): Promise<PointCloudOctree> {
-    return loadPOC(url, getUrl, xhrRequest).then(geometry => new PointCloudOctree(this, geometry));
+  async loadPointCloud(url: string, getUrl: GetUrlFn, xhrRequest = (input: RequestInfo, init?: RequestInit) => fetch(input, init)): Promise<PointCloudOctree> {
+    if (url === 'cloud.js') {
+      return await loadPOC(url, getUrl, xhrRequest).then((geometry) => {
```

Comment:
`return await` can almost always be replaced by just `return`.
The only cases where a `return await` may make sens are:

When returning some operation that this await is just part of, for example:
```typescript
 return await getNumber() * 2 + await getOtherNumber();
```
But the "await" here is not really applied to the full return statement, but just to `getNumber()`. 

When you want to handle an exception in the promise you return : 
```typescript
try {
  return await getValue();
} catch(e) {
  return FALLBACK_VALUE;
}
```